ui/components/app/connected-accounts-permissions/connected-accounts-permissions.js

```diff
@@ -59,9 +59,9 @@ const ConnectedAccountsPermissions = ({ permissions }) => {
               key={`connected-permission-${idx}`}
               className="connected-accounts-permissions__list-item"
             >
-              <CheckBox
-                checked
-                disabled
+              <Checkbox
+                isChecked
+                isDisabled
```

Comment:
Can you use the `label` prop on the Checkbox here? and remove the label? Also double check to see if you can remove the styles associated with `connected-accounts-permissions__checkbox` class